modules/insights-operator-using-filters-on-advisor-recommendations.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="insights-operator-using-filters-on-advisor-recommendations_{context}"]
+= Using filters to remove unnecessary results from Insights advisor recommendations
+
+As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. You might do this to reduce the number of reported recommendations or to concentrate on your highest priority issues. 
+
+Use the following example to select from several categories of recommendations to hide unneeded results.
+
+NOTE: The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the primary and secondary filters. 
+
+.Prerequisites
+You are logged into the Red Hat Hybrid Cloud Console.
+
+.Procedure
+. Navigate to the https://console.redhat.com/openshift/insights/advisor/recommendations[Red Hat Hybrid Cloud Console > OpenShift > Advisor recommendations] page.
+. Using the filter input fields above the list of recommendations, select the *Category* primary filter.
+. Expand the secondary filter dropdown and check the box next to each category of recommendation you want to view. Leave unnecessary categories unchecked.
+. Optionally, add additional filters to further refine the list.
+
```

Comment:
We can make this procedure generic and simple, rather sepcifying any specific filter like Category.

. From the *filter types* list that is present above the list of recommendations, select a filter.
. Set the value of the filter by selecting the relevant options from the drop-down list.
. Optional: Apply additional filters by performing the same steps to further refine the recommendations list.